plugin-server/src/worker/ingestion/property-definitions-cache.ts

```diff
@@ -36,7 +37,8 @@ export class PropertyDefinitionsCache {
     }
 
     async initialize(teamId: number, db: DB): Promise<void> {
-        const properties = await db.postgresQuery(
+        const properties = await db.postgres.query(
+            PostgresUse.COMMON_READ,
```

Comment:
would likely be safer to do `COMMON_WRITE` initially to avoid potential problems around: 
1. read
2. do I need to update
3. expensive write path